nginx.md

```diff
@@ -0,0 +1,190 @@
+description: Nginx can be very powerful, but it takes a little extra configuration with Pico - Let's break it down.
+toc:
+  getting-started: Getting Started
+  general-server-configuration: General Server Configuration
+  denying-access-to-picos-internal-files: Denying Access to Pico's Internal Files
+  php-configuration: PHP Configuration
+  url-rewriting:
+    _title: URL Rewriting
+    pico-in-document-root: Pico in Document Root
+    pico-in-a-subfolder:
+      _title: Pico in a Subfolder
+      regex-and-nginxs-processing-order: Regex and Nginx's Processing Order
+  example-configuration: Example Configuration
+  advanced-configuration-tips:
+    _title: Advanced Configuration Tips
+    modular-pico-config: Modular Pico Config
+nav-url: /docs/
+---
+
+[Nginx](https://www.nginx.com/) has quickly become a solid contender in the realm of web servers.  While most of the web still uses Apache for serving web content, we acknowledge that a growing portion of our users may be interested in deploying Pico on Nginx instead of Apache.  Deploying Pico on Apache at the moment is relatively painless, with Pico's `.htaccess` file providing almost all the configuration necessary.
+
+Unlike Apache, Nginx uses a ["Centralized" configuration](https://www.digitalocean.com/community/tutorials/apache-vs-nginx-practical-considerations#distributed-vs-centralized-configuration).  Apache scours every folder it hosts in search of `.htaccess` files.  These files hold extra configuration options, often deployed with hosted software to make their setup easy.  Pico is no exception to this, and in most cases, Pico's `.htaccess` provides all the settings required to get it up and running without user interaction.
+
+This "Distributed" configuration can make it hard to understand what's really going on behind the scenes.  Apache's configuration can be spread out into as many folders as your webapps (or your own content) provide.
+
+In comparison, Nginx's "Centralized" configuration is all located in one place.  On a Linux server, this configuration would usually be located in `/etc/nginx`, but this may vary by distro and OS.  While [configuration of Nginx](https://www.nginx.com/resources/wiki/start/) as a whole is out of the scope of this document, we hope to provide you with enough information to get over any hurdles you may encounter.
+
+If you are migrating from Apache, [this article](https://www.digitalocean.com/community/tutorials/apache-vs-nginx-practical-considerations) (also linked above) and [this tutorial](https://www.digitalocean.com/community/tutorials/how-to-migrate-from-an-apache-web-server-to-nginx-on-an-ubuntu-vps) provide an excellent overview on the differences between the two and the changes you'll have to make.
+
+## Getting Started
+
+While the [example]({{ site.github.url }}/docs#nginx-configuration) provided on the previous page is a good starting point, it will likely not be enough to fully configure your Pico installation.
```

Comment:
I don't think that this is true, otherwise the provided example would be wrong or seriously incomplete :wink:

The difference rather is that the example provided here is more comprehensive, also covers the relevant general webserver configuration, explains the *Why* and weighs alternatives. :smiley: